lib/fa2/common/tzip12.datatypes.jsligo

```diff
@@ -0,0 +1,37 @@
+export type tokenMetadataData = {
```

Comment:
tokenMetadataData seems weird, it should be tokenMetadata only ?

Reply:
No, tokenMetadata  is different from tokenMetadataData 
There is a field data on the tokenMetadata